### ISSUE TITLE
It's possible to donate?

### ISSUE THREAD

**Issue by webolot** [pwr-Solaar/Solaar#1297]
Hi, 

I'm a SysAdmin, not a developer. I can't contribute coding, but I can pay a few beers (or cofee, or tea, or...) to thank for this really usefull project!

It's possible to donate? How?

If not, I can contribute translating Solaar to catalan language if you want.

Thank's for your work!

**Comment by pfps** [comment]
We are not set up to handle donations.

Translation to Catalan would be useful.  More useful would be better documentation.

**Comment by mserramanxa** [comment]
when you got 2 minutes, can you create a pull request for add the catalan language to the next release? thank's

**Comment by webolot** [comment]
when you got 2 minutes, can you create a pull request for add the catalan language to the next release? thank's

**Comment by pfps** [comment]
My apologies.  I thought that I had followed up on this, but I didn't.
The usual process would be that you create a PR that then is approved, but if you would prefer that I do it that's fine as well.

**Comment by mschilli87** [comment]
> Sure, but I'm not sure if I can create a pull request via browser (the button CREATE PULL REQUEST is disabled) and I'm not very comfortable using github.

@webolot: To create a PR yourself you need to first fork the repo using the button in the top right. Then you apply the changes to your fork and the PR button will become active.

See [this article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) for additional information.

**Comment by webolot** [comment]
> My apologies. I thought that I had followed up on this, but I didn't. The usual process would be that you create a PR that then is approved, but if you would prefer that I do it that's fine as well.

No apologies needed! Thank's for your time!

**Comment by webolot** [comment]
> > Sure, but I'm not sure if I can create a pull request via browser (the button CREATE PULL REQUEST is disabled) and I'm not very comfortable using github.
> 
> @webolot: To create a PR yourself you need to first fork the repo using the button in the top right. Then you apply the changes to your fork and the PR button will become active.
> 
> See [this article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) for additional information.

Oh yeah! Clear and simple! I think I just created a pull request. Thank's for the link!

**Comment by pfps** [comment]
Yes indeed.  I merged the PR into Solaar so the Catalan translation should show up in the next release.  

Modified translations don't automatically take effect when you clone the github code, but you can enable them by running ./tools/po-compile.sh
